test/e2e/singleNodeQuery.go

```diff
@@ -54,6 +55,7 @@ func main() {
 	flag.IntVar(&testCfg.singleNodeNum, "single_node_num", 1, "Number of single node set requested from redis, default to 1")
 	flag.DurationVar(&testCfg.singleNodeInterval, "single_node_interval", 1*time.Second, "Query interval of single node, default to 1s")
 	flag.IntVar(&testCfg.batchNodeNum, "batch_node_num", 10, "Number of batch node, default to 0s")
+	flag.StringVar(&testCfg.remoteRedisPort, "remote_redis_port", "7379", "Remote redis port, if not set, default to 7379")
```

Comment:
redisport should be part of rmsclient.Config, which is shared by multi component. testConfig is a local struct, only be used for test related parameter.

Reply:
After team discussion, redisPort is still in testConfig here.